packages/web/index.js

```diff
@@ -38,7 +39,8 @@ module.exports = (neutrino, opts = {}) => {
     },
     style: {
       hot: opts.hot !== false,
-      extract: opts.extract || process.env.NODE_ENV === 'production' && {}
+      extract: extract && {},
+      css: extract && { allChunks: true }
```

Comment:
Like `ignoreOrder`, `allChunks` is an option for `opts.style.extract.plugin`, not `opts.style.css`.

Reply:
🤦‍♂️